Packs/CommonScripts/ReleaseNotes/1_10_40.json

```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":"The script: GenerateAsBuilt moved to *CommonScripts* pack. Make sure to update the *CommonScripts* pack to the latest version in order to use this script."}
```

Comment:
This is not the breaking change, the breaking change should be in the actual pack we removed the script from.

Reply:
I did as follows in the given example: https://github.com/demisto/content/pull/23434/files
As far as i know the "breaking change" should be a json in the release notes folder of the pack.
The original Packs/GenerateAsBuilt does not even have a release notes folder.
What am i missing?

Reply:
@samuelFain Yasmin is right. This should be in the original pack (GenerateAsBuilt).

Reply:
It does not have RN folder because it was the first and only version. once it has a change the folder should be created. The reason you saw it in the example is that it was moved *from* CommonScript to another folder.
The breaking change should be on the pack we broke.

Reply:
Got it.

Reply:
Fixed.